### DISCUSSION TITLE
Availability of the validation dataset

### DISCUSSION THREAD

**Discussion by dvryaboy** [facebookresearch/esm#563] [ESM Metagenomic Atlas]
Hi folks, thanks for making this amazing resource available. In the paper, you describe the process you used to separate out the validation data set as follows:
> UniRef50, April 2021 version, is used for the training of ESM models. We partition the training dataset
by randomly selecting 0.5% (≈250,000) sequences to form our validation set. The training set has
sequences removed via the procedure described in (54). MMseqs search (–min-seq-id 0.5
–alignment-mode 3 –max-seqs 300 -s 7 -c 0.8 –cov-mode 0) is run using the train set as query database
and the validation set as target database. All train sequences which match a validation sequence with
50% sequence identity under this search are removed from the train set.

Is it possible to make the training and validation datasets prepared this way available, or to share the random seed and command you used to select the validation set? This would help avoid any training set contamination concerns for folks trying to reproduce and iterate on this work.
